File: SimpleOrder/Models/InvoiceLineItem.cs
namespace SimpleOrder.Models
{
    public class InvoiceLineItem : EntityBase
    {
        public int LineNumber { get; set; }

        public string ItemNumber { get; set; }

        public string Description { get; set; }

        public double UnitPrice { get; set; }

        public int Quantity { get; set; }

        public double Discount { get; set; }

        public double SubTotal { get; set; }
    }
}

File: SimpleOrder/ClientApp/src/components/invoice/InvoiceLineItem.js
import React, { Component } from 'react';
import { formatCurrency } from '../controls/Shared';

export class InvoiceLineItem extends Component {
    constructor(props) {
        super(props);
        this.state = this.props.state;
    }

    render() {
        let item = this.state;
        return (
            <div className="invoiceLineItem">
                <div className="itemNumber">{item.ItemNumber}</div>
                <div className="description">{item.Description}</div>
                <div className="unitPrice">{formatCurrency(item.UnitPrice)}</div>
                <div className="quantity">{item.Quantity}</div>
                <div className="subTotal">{formatCurrency(item.SubTotal)}</div>
            </div>
        );
    }
}
